tests/storage_test.go

```diff
@@ -402,6 +410,23 @@ var _ = Describe("Storage", func() {
 					}
 				})
 
+				Context("Without the HostDiskVMIAccess feature gate enabled", func() {
+					BeforeEach(func() {
+						tests.DisableFeatureGate(virtconfig.HostDiskGate)
+					})
+
+					It("Should fail to start a VMI", func() {
+						diskName := "disk-" + uuid.NewRandom().String() + ".img"
+						diskPath := filepath.Join(hostDiskDir, diskName)
+						vmi = tests.NewRandomVMIWithHostDisk(diskPath, v1.HostDiskExistsOrCreate, "")
+						tests.RunVMIAndExpectLaunch(vmi, 30)
+						virtClient, err := kubecli.GetKubevirtClient()
+						Expect(err).ToNot(HaveOccurred())
+						_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+						Expect(err).To(HaveOccurred())
```

Comment:
Can we check if the expected cause is included in the error?

Reply:
Good point, done.